README.md

```diff
@@ -108,6 +109,43 @@ module.exports = function (src) {
 };
 ```
 
+## Twig tag
+
+Aside from the `webpack_asset` twig function, you can also use the `webpack` tag to specify one or more entry points in
+a much more elegant fashion. The syntax of this tag works like this:
+
+```twig
+{% webpack <type: css|js> <list-of-javascript-files> %}
+    {{ asset }}
+{% endwebpack %}
+```
+
+If you want to include javascript files, simply do this:
+```twig
+{% webpack js
+   '@AppBundle/file1.js'
+   '@AppBundle/file2.js'
+%}
+    <script src="{{ asset }}"></script>
+{% endwebpack %}
+```
+
+The same method can be applied for CSS files.
+```twig
+{% webpack css '@AppBundle/file1.js' %}
+    <link rel="stylesheet" href="{{ asset }}">
+{% endwebpack %}
+```
+
+Note that in the CSS example, we're still referencing javascript files. _This is not a mistake._ Webpack extracts
+referenced CSS files from javascript (or any subset of it) files and places them in separate css files - if the bundle
```

Comment:
`... from javascript, or any subset of it, files and ...` this reads a bit weird.

What about removing the part between the parenthesis? It's not really useful info imo